packages/sdk/src/application/ApplicationClient.ts

```diff
@@ -438,31 +438,55 @@ export class ApplicationClient extends APIResource {
         await this._client.post<RestartPlaygroundConversationResponse>(
           queryString,
         );
-      return response;
+      return Promise.resolve(response);
     } catch (error) {
       return Promise.reject(error);
     }
   }
 
-  async chat(props: ChatRequest): Promise<ChatResponse> {
-    const { ownerId, appId, ...payload } = props;
+  async chat(
+    props: ChatRequest,
+    stream: boolean = true,
+  ): Promise<ChatResponse | unknown> {
```

Comment:
@thewbuk SDK can't return `unknown` you need to overload this 

- When stream is enable then it's any or ReadableStream
- When stream is disable you return the normal one